runtime/src/bank.rs

```diff
@@ -416,6 +416,8 @@ pub struct BankRc {
 
     /// Current slot
     pub(crate) slot: Slot,
+
+    pub(crate) slot_id_generator: Arc<AtomicU64>,
```

Comment:
Used to distinguish between banks with the same slot, see description here: https://github.com/solana-labs/solana/pull/17471/files#r645937617